src/datepicker/datepicker.module.ts

```diff
@@ -3,6 +3,9 @@ import { NgModule } from "@angular/core";
 import { CommonModule } from "@angular/common";
 import { DatePicker } from "./datepicker.component";
 
+import * as languages from "flatpickr/dist/l10n/index";
+export default languages;
```

Comment:
We don't want to default export `languages` ... does `export languages;` or `export { languages };` not work?

Reply:
They both give errors.

`export languages;` gives the following error:
```
Declaration or statement expected
```

`export { languages };` gives: 
```
...datepicker.module has no default export.
```

Reply:
Hmm ... `export const languages = flatpickrLanguages;` perhaps? Weird that the object export wouldn't work

Reply:
That gives me `Individual declarations in merged declaration 'languages' must be all exported or all local.`

Reply:
Okay now it's just getting silly

Reply:
Let's skip that for now ... ideally it should work with the i18n service down the road, but we do need _something_ working in the short term